R/epidatacall.R

```diff
@@ -186,6 +186,7 @@ fetch_args_list <- function(
 #' @return
 #' - For `fetch`: a tibble or a JSON-like list
 #' @export
+#' @include cache.R
```

Comment:
Does this ensure the cache_environ var is present?

Reply:
exactly, its what generated the collation